plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaRecordSet.java

```diff
@@ -20,6 +20,7 @@
 import io.trino.decoder.DecoderColumnHandle;
 import io.trino.decoder.FieldValueProvider;
 import io.trino.decoder.RowDecoder;
+import io.trino.plugin.kafka.clients.consumer.KafkaConsumerFactory;
```

Comment:
I am not sure if we need this repackaging.

@losipiuk @lukasz-walkiewicz WDYT?

Reply:
If I can add my reasoning behind this.

Since we are planning on adding more security protocols e.g. `SASL_PLAINTEXT`, `SASL_SSL` which means new clients will need to be added and implement their assigned interfaces. 
To avoid having too many files in the same package, I think separating by client type (consumer/producer/admin) makes it more organized similarly to what Apache Kafka [did](https://github.com/apache/kafka/tree/trunk/clients/src/main/java/org/apache/kafka/clients) .

Reply:
I would suggest to refrain from this refactor now. And let's do it as separate change after this PR lands. It is a bit confusing that we move class around for this change.

Reply:
Np. I moved all classes under `io.trino.plugin.kafka.clients` to `io.trino.plugin.kafka`